terms.jsonc

```diff
@@ -125,7 +125,7 @@
   "SSH",
   ["wi[- ]?fi", "Wi-Fi"],
   "XML",
-  "YAML",
+  ["(?<![\\.-])yaml\\b", "YAML"],
```

Comment:
It's funny because I had to code the exact sane things for jargonLint

Reply:
I'm surprised by the `-`. Could you share an example ?

Reply:
Oh, that's right. But then you should limit by looking for a dot after yaml, no?

Maybe you should also support underscore, no?

Thanks for your feedback, I will support it in jargonLint

Reply:
Great for your project !